bin/lib/Logos/Class.pm

```diff
@@ -25,12 +25,15 @@ sub new {
 ##################### #
 # Setters and Getters #
 # #####################
+
+# Such as 'UIView'
```

Comment:
Shouldn't this be something like `'customView' from 'UIView * customView'`?

Reply:
I don't think so but I'll double check

Reply:
I'm correct, it's the name of a type, such as `UIView` from `%hook UIView`. [For example](https://github.com/theos/logos/blob/master/bin/logos.pl#L249-L263)

Reply:
Oh right, I didn't notice I was looking at Class.pm